team/_posts/2017-05-14-peraza-julio.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: member
+title: Julio A Peraza
+position: Research Affiliate
+department:
+handle: jperaza
+nickname: Julio
+science_names: [Peraza-Goicolea JA]
+image: /assets/images/team/julio-peraza.jpg
+alumni: false
+
+# social
+cv: https://julioaperaza.github.io/cv/
+email: jpera054@fiu.edu
+github: JulioAPeraza
+orcid: 0000-0003-3816-5903
+osf:
+publons:
+researchgate:
+impactstory:
+scholar: 3mxI9IIAAAAJ&hl
+site: https://julioaperaza.github.io
+twitter:
+---
```

Comment:
Can you add a blurb about yourself here?

Reply:
ok